modules/react-components/src/components/grid/resource-grid/resource-grid.tsx

```diff
@@ -97,6 +97,27 @@ export const ResourceGrid: FunctionComponent<
             wrapperClassName
         );
 
+        const numberOfPlaceholderCards: number = 4;
+
+        if (isLoading) {
+            return (
+                <Card.Group style={ { marginBottom: "3rem" } }>
+                    {
+                        [ ...Array(numberOfPlaceholderCards) ].map((_, index) => {
+                            return (
+                                //TODO: Add placeholder style classes.
+                                <Card key={ index } style={ { boxShadow: "none", width: "220px" } }>
+                                    <Placeholder>
+                                        <Placeholder.Image style={ { height: "230px" } } />
+                                    </Placeholder>
+                                </Card>
+                            );
+                        })
```

Comment:
Can't we just use a for loop here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/13fc653e28ee815a2f5323a1157155f5b1e4b3da